src/main/clm_initializeMod.F90

```diff
@@ -606,6 +614,14 @@ subroutine initialize2( )
        call crop_inst%initAccVars(bounds_proc)
     end if
 
+    ! use_fates_spitfire is assigned an integer value in the namelist
+    ! see bld/namelist_files/namelist_definition_clm4_5.xml for details
+    if (use_fates_spitfire > 1) then
```

Comment:
Just call the generic one rather than the specific one

Reply:
Same [comment](https://github.com/ESCOMP/CTSM/pull/991#discussion_r415424002).